sdk/tables/azure-data-tables/inc/azure/data/tables/models.hpp

```diff
@@ -117,6 +117,16 @@ namespace Azure { namespace Data { namespace Tables {
        */
       std::vector<Models::Table> Tables;
 
+      /**
+       * @brief Construct a new Query Tables Paged Response object.
+       *
+       * @param tableServiceClient Table service client.
+       */
+      QueryTablesPagedResponse(std::shared_ptr<TableServiceClient> tableServiceClient)
```

Comment:
Can this be private? If it is only the `Azure::Data::Tables::TableServiceClient` and `Azure::Core::PagedResponse<QueryTablesPagedResponse>` who can/should/expected to create this, we should, I think.

Reply:
done